taskweaver/session/session.py

```diff
@@ -21,6 +22,7 @@ def _configure(self) -> None:
         self.code_interpreter_only = self._get_bool("code_interpreter_only", False)
         self.max_internal_chat_round_num = self._get_int("max_internal_chat_round_num", 10)
         self.plugin_only_mode = self._get_bool("plugin_only_mode", False)
+        self.cli_only_mode = self._get_bool("cli_only_mode", False)
```

Comment:
change to only one config mode?

Reply:
merged as one config parameter `session.code_exec_mode`